### PR TITLE
Update welcome page

### PR DESCRIPTION
**Pull request by originated**
Fixes for #2892

### CODE REVIEW

**Comment by roadscape** [comment]
It would be better to keep the commit history from this community PR, mind if I rebase/force-push on this branch?

**Comment by roadscape** [comment]
Could not rebase, it looks as though changes have been made to the parent since this branch was created.

**Comment by roadscape** [comment]
Ok conflicts were not too bad; all resolved, rebased, and made 2 minor corrections.

File: src/app/help/en/welcome.md [hunk 1]
```diff
@@ -39,27 +39,24 @@ Below that is a list of recommended users to follow, a collection of other resou
 - <a href="#Password_Security">Password Security</a>
 - <a href="#Earning_on_Steemit">Earning on Steemit</a>
 
-### <a href="#Helpful_Posts_from_Steemit_Users">Helpful Posts from Steemit Users</a>
```
Comment (plink01001) [inline src/app/help/en/welcome.md]: This line should be added back

**Pull request merged**